src/app/components/ThemeProvider.tsx

```diff
@@ -95,8 +95,6 @@ export default function ThemeProvider({
               font-family: inherit;
             }
             .rt-table {
-              display: block;
-              width: 0;
             }
```

Comment:
Can probably remove the selector as well then?

Reply:
good point yes